### PR TITLE
[stable/gcp-night-king] Add new Helm Chart for GCP Night King service

### PR DESCRIPTION
**Pull request by itamaro**
Did my best following contribution guidelines & best practices :-)

The service behind the Chart: https://github.com/itamaro/gcp-go-night-king

### CODE REVIEW

**Comment by unguiculus** [comment]
Please make list indentation consistent. I'd suggest you always indent by two spaces.

**Comment by itamaro** [comment]
> Please make list indentation consistent. I'd suggest you always indent by two spaces.

fixed

File: stable/gcp-night-king/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "nightking.name" -}}
```
Comment (unguiculus) [inline stable/gcp-night-king/templates/_helpers.tpl]: The template namespace should be the same a the chart name: `gcp-night-king`.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/gcp-night-king/templates/_helpers.tpl (hunk 1 above)
Comment (itamaro) [reply stable/gcp-night-king/templates/_helpers.tpl]: right. fixed.

**Comment by itamaro** [comment]
Any other concerns I should address?

**Review by unguiculus — APPROVED** [review]

**Comment by itamaro** [comment]
Thanks for approving this @unguiculus :-)

I've squashed and rebased, what else do I need to do to get this merged?

I don't know what it means "Not in tide pool."...

**Comment by unguiculus** [comment]
You didn't have to squash. That's done upon merge anyways. Only my lgtm is missing.

**Comment by itamaro** [comment]
> You didn't have to squash. That's done upon merge anyways. Only my lgtm is missing.

cool, thanks :-)

**Pull request merged**